App/Views/PcbSinglePage.xaml.cs

```diff
@@ -11,11 +23,40 @@ public PcbSingleViewModel ViewModel
         get;
     }
 
+    //private static BitmapImage dataMatrixImageSource = new BitmapImage();
+    //private static ImageBrush dataMatrixImageBrush = new ImageBrush();
+
     public PcbSinglePage()
     {
         ViewModel = App.GetService<PcbSingleViewModel>();
         InitializeComponent();
         DataContext = ViewModel;
         dateOfFailure.Date = new DateTime(2023, 04, 23);
+        GenerateBarcode();
+    }
+
+    private async void OnPrintButtonClicked(object sender, RoutedEventArgs e)
+    {
+        /*var data = new PageData(this, dataMatrixRectangle);
+        var rect = (Rectangle)data.Rectangle.FindName("dataMatrixRectangle");
+        rect.Fill = dataMatrixImageBrush;
+        var page = data.Page;
+        var pageTest = new Page();
+        var layoutControl = new Grid();
+        //layoutControl.Children.Add(uiElementGrid1);
+        layoutControl.Children.Add(uiElementGrid2);
+        pageTest.Content = layoutControl;*/
+        
+        IPrintService printSerivce = new PrintService();
+        await printSerivce.Print(Test); //PcbSinglePageContent
```

Comment:
Beim versuch zu drücken ist das Einzige UI Element das angezeigt wird, der 2d Code. Der aktuelle Stand auf Dev mit der gesamten Settingsseite ist meiner Meinung nach besser zu Vorführung geeignet. Als der Stand dieses Branches.

Mein Voschlag wäre in Zeile 51 "        await printSerivce.Print(PcbSinglePageContent)" schreiben, um mehr zuzeigen. Das das vom FOrmat noch nicht passt, ist dann erstmal irrelevant.

Reply:
Ich zitiere aus meinem Kommentar des PR oben:
"Known-Bug: Es werden nicht alle UI Elemente zum Drucken angezeigt, muss noch gefixt werden."

Änderung wie gewünscht vorgenommen.